README.md

```diff
@@ -0,0 +1,90 @@
+# Instructions to create a new plugin
+
+The process to create a new `plugin` able to manage a new capability for the `halkyon` operator is pretty forward 
+and consists in a few steps :
+
+- Git clone the project `https://github.com/halkyonio/plugin-example-capability` and rename it 
+- Rename the `module name` as defined within the `go.mod` file to use your package name
+- Find and replace the `example` word with the name of your `plugin`, resource, ...
```

Comment:
That's not really detailed enough, not very clear…

Reply:
Fixed